frontend/src/api/ApiCaller.tsx

```diff
@@ -128,10 +128,16 @@ export class BackendAPICaller {
         return result.content
     }
     async postMission(echoMissionId: number, robotId: string, startTime: Date) {
+        const assetCode = sessionStorage.getItem('assetString')
```

Comment:
Is it OK to get asset like this or should it be given as an argument in postMission instead?